src/class/msc/msc.h

```diff
@@ -284,8 +284,8 @@ typedef struct ATTR_PACKED
 {
   uint8_t data_len;
   uint8_t medium_type;
-  bool write_protected : 1;
   uint8_t reserved : 7;
+  bool write_protected : 1;

```

Comment:
this is not important, but it would be nice if you change the write_protected to uint8_t instead of bool, make it clearer that WP is part of 8-bit bitfiled with above reserved.